packages/web/src/components/premium-content-purchase-modal/components/PurchaseSummaryTable.tsx

```diff
@@ -44,21 +44,19 @@ export const PurchaseSummaryTable = ({
       {existingBalance ? (
         <div className={styles.row}>
           <span>{messages.existingBalance}</span>
-          <span>{`-${messages.price(
-            formatUSDCWeiToUSDString(existingBalance)
-          )}`}</span>
+          <span>{`-${messages.price(formatPrice(existingBalance))}`}</span>
         </div>
       ) : null}
       <Text className={styles.row} variant='title'>
         <span>{messages.youPay}</span>
         <span className={styles.finalPrice}>
           {existingBalance ? (
             <>
-              <s>{messages.price(formatUSDCWeiToUSDString(basePrice))}</s>
-              {messages.price(formatUSDCWeiToUSDString(amountDue))}
+              <s>{messages.price(formatPrice(basePrice))}</s>
```

Comment:
ngl I forgot the `<s>` tag was a thing!

TIL, kinda interesting: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/s#accessibility_concerns

Reply:
Just need a `<marquee>` now to round out the OG HTML tag bingo!

More seriously, I did read the accessibility thing. I can add the content helper to get us closer to being accessible. Or there may be some other `aria` properties we could use to indicate that it's out of date information.

Reply:
this random guy on google said use `<del>` https://pauljadam.com/demos/css-line-through-del-ins-accessibility.html

Reply:
<del> it is!